src/components/v2/appDetails/k8Resource/nodeType/nodeType.util.ts

```diff
@@ -38,3 +39,18 @@ export const getNodeStatus = (node: iNode) => {
     }
     return ''
 }
+
+export const getFilteredPodStatus = (podStatusObj: NodePodStatus) => {
+    const podStatusKeys = Object.keys(podStatusObj)
+
+    if (podStatusKeys.length > 2 && podStatusObj['running'] === 0) {
```

Comment:
let's add a comment what does this 2 signify

Reply:
existing code 
no idea about it